src/main/kotlin/race/Car.kt

```diff
@@ -1,15 +1,18 @@
 package race
 
-class Car(private var position: PositiveNumber) {
-    val movedPosition: PositiveNumber
-        get() = position
+class Car(
+    private val name: Name,
+    private var position: PositiveNumber,
+) {
+    val progress: Int
+        get() = position.value
+    val displayName: String
+        get() = name.value
```

Comment:
```suggestion
    val progress: Int
        get() = position.value
    val displayName: String
        get() = name.value
```
이렇게 사용한 이유는 외부에서 사용할때, `car.position.value` -> `car.progress` 뎁스를 줄이기 위해 사용해보았습니다~

Reply:
position을 생성자로 받을 때는 PositiveNumber을 받고, 노출은 progress 로써 Int를 노출한다면, data class를 활용해도 충분히 괜찮다고 생각해요.
제가 제안드린 것은, var인 PositiveNumber를 외부에 공개하더라도, setter만 공개하지 않는 방법에 더해, 초기 PositiveNumber를 생성자로 주입해줄 수 있는 형태를 만들 수 있게 하기 위해 data 키워드를 빼보라는 의미었어요!
```kotlin
class Car(
    val name :Name,
    position: PositiveNumber,
) {
    var position: PositiveNumber = position
        private set
}

```
현 코드라면, data 키워드를 사용하는 게 합리적이란 생각이 드네요 :)

Reply:
4단계 진행 초반에 `private set` 적용했다가,  `progress`로 통해 뎁스를 줄이기 위해서 사용하기에는 애매하더라구요! ㅜㅜ
이부분은 좀 더 코틀린 스러운것이 무엇인지 고민해보겠습니다!